functions/slack/app/go.mod

```diff
@@ -0,0 +1,13 @@
+module github.com/GoogleCloudPlatform/golang-samples/functions/slack/app
+
+go 1.11
+
+require (
+	cloud.google.com/go v0.44.3
+	github.com/GoogleCloudPlatform/golang-samples v0.0.0-20190819213247-9dcbc6f1dc40
```

Comment:
Same comment as OCR - we should get rid of this dependency, even though it means duplicating code to check for `GOLANG_SAMPLES_PROJECT_ID`.